drivers/SmartThings/matter-appliance/src/matter-extractor-hood/init.lua

```diff
@@ -14,21 +14,74 @@
 
 local capabilities = require "st.capabilities"
 local clusters = require "st.matter.clusters"
+local utils = require "st.utils"
 
 local EXTRACTOR_HOOD_DEVICE_TYPE_ID = 0x007A
+local ON_OFF_LIGHT_DEVICE_TYPE_ID = 0x0100
+local ON_OFF_LIGHT_SWITCH_DEVICE_TYPE_ID = 0x0103
```

Comment:
We don't technically support this device type since it requires bindings, but some manufacturers use this to indicate a light switch device while still implementing the clusters as server (which means bindings aren't required and we can support it).

Is this added here just to keep parity with this special case handling in the `matter-switch` driver, or is there an extractor hood device that actually has this device type?

Reply:
Additionally, if we are going to check for this special case, we need to check that this device implements the on off cluster as server when accounting for these endpoints. This is done like this in the matter-switch driver: https://github.com/SmartThingsCommunity/SmartThingsEdgeDrivers/blob/ab626b60b7f92784886e6e56b5a12c5870e70c1c/drivers/SmartThings/matter-switch/src/init.lua#L541

Reply:
The STMicro extractor hood uses this device type. Adding that check is a good call!